### PR TITLE
[WWST-7349] Added fingerprint for Leak Gopher water leak sensor

### PR DESCRIPTION
**Pull request by MWierzbinskaS**
@SmartThingsCommunity/srpol-pe-team 
@greens 
Could you please take a look at the code?

### CODE REVIEW

File: devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy [hunk 1]
```diff
@@ -31,6 +31,7 @@ metadata {
 		fingerprint mfr: "0086", prod: "0002", model: "007A", deviceJoinName: "Aeotec Water Leak Sensor" //EU //Aeotec Water Sensor 6
 		fingerprint mfr: "0086", prod: "0202", model: "007A", deviceJoinName: "Aeotec Water Leak Sensor" //AU //Aeotec Water Sensor 6
 		fingerprint mfr: "000C", prod: "0201", model: "000A", deviceJoinName: "HomeSeer Water Leak Sensor" //HomeSeer LS100+ Water Sensor
+		fingerprint mfr: "0173", prod: "4C47", model: "4C44", deviceJoinName: "Leak Gopher Water Leak Sensor" //Leak Intelligence Leak Gopher Z-Wave Leak Detector
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy:35]: @MWierzbinskaS Please, could You add a raw description in a comment after the fingerprint ?

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy [hunk 2]
```diff
@@ -58,7 +59,7 @@ metadata {
 }
 
 def initialize() {
-	if (isAeotec() || isNeoCoolcam() || isDome()) {
+	if (isAeotec() || isNeoCoolcam() || isDome() || isLeakGopher()) {
 		// 8 hour (+ 2 minutes) ping for Aeotec, NEO Coolcam, Dome
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy]: aaaand now also for LeakGopher 😺 (same for below else...if statement change)

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy (hunk 2 above)
Comment (MWierzbinskaS) [reply devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy]: Done :)

File: devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy (hunk 1 above)
Comment (MWierzbinskaS) [reply devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy:35]: Done :)

**Review by KKlimczukS — APPROVED** [review]

**Review by PKacprowiczS — APPROVED** [review]

**Pull request merged**